test/regress/regress0/fp/issue7002.smt2

```diff
@@ -0,0 +1,4 @@
+; EXPECT: sat
+(set-logic ALL)
+(assert (= 0.0 (fp.to_real (_ NaN 8 24))))
+(check-sat)
```

Comment:
Let's also add the regression to CMakeLists.txt.

Reply:
Whoops :D